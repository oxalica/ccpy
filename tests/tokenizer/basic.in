if iff aaa123+321aaa # comment inside
.... ... .. .
True False None
'123' "2
34"
'a\n"b\'' "\"3'9\\"
