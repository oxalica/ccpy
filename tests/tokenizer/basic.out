Keyword("if")
Name("iff")
Name("aaa123")
Symbol("+")
Integer("321")
Name("aaa")
Newline
Symbol("...")
Symbol(".")
Symbol("...")
Symbol(".")
Symbol(".")
Symbol(".")
Newline
Keyword("True")
Keyword("False")
Keyword("None")
Newline
String("123")
String("2\n34")
Newline
String("a\n\"b'")
String("\"3'9\\")
Newline
End
